packages/terra-consumer-nav/src/components/help-button/HelpButton.scss

```diff
@@ -0,0 +1,17 @@
+@import '../../variables';
+
+:local {
+  .help-button {
+    background: var(--terra-consumer--help-button-background, #fff);
+    border-radius: 20px;
+    bottom: 15px;
+    box-shadow: 0 1px 1px #666;
+    padding: 10px;
+    position: absolute;
```

Comment:
I think so, this was temporary to get it on the screen, @Mahmud-Khan is working on this component and could provide better input on the css required here.